spec/uvobot/notifications/slack_notifier_spec.rb

```diff
@@ -32,7 +32,7 @@
 end
 
 def params(message)
-  ['slack.com', payload_json_string(message)]
+  ['slack.com', body: payload_json_string(message)]
```

Comment:
Httparty
# Simple post with full url and setting the body
Foo.post('http://foo.com/resources', body: {bar: 'baz'})

Ocakavam tam ze double dostane volanie s parametrami vo formate aky berie aktualne http client.